oblib/data_model.py

```diff
@@ -493,6 +505,13 @@ def _toJSON(self):
         aspects["concept"] = self.concept
         if self.unit is not None:
             aspects["unit"] = self.unit
+            if self.unit == "pure":
+                aspects["decimals"] = "0"
```

Comment:
why force `decimals` to 0?  The spec doesn't appear to require this.  What if a user was trying to report 10% which would be `0.10` as a pureItemType?

Reply:
This is a holdover from an earlier version of the code, and I agree is not needed anymore. I'll remove it.

Reply:
Removed.